.eslintrc.json

```diff
@@ -65,34 +64,25 @@
     {
       "files": ["*.ts", "*.tsx"],
       "extends": ["plugin:@nx/typescript"],
-      // TODO:CHECKOUT-9228 Fix lint errors after nx upgrade to 19.8.9
       "rules": {
```

Comment:
Each rule is checked manually, and those that remain are either costly to refactor or related to class components, which we will refactor soon.